content/en/docs/setup/install/multicluster/virtual-machine/index.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Installing Istio for virtual machine connectivity
```

Comment:
I am not sure I understand this doc. Why would I follow this doc rather than https://deploy-preview-7248--preliminary-istio.netlify.app/docs/examples/virtual-machines/single-network/? It seems to have considerable overlap

Reply:
I think having two docs for the same purpose will confuse our users (and me!)

Reply:
@rcaballeromx and @frankbu and have requested I delete the install section from the other docs, delete one of the virtual machine integration docs entirely, and include only the example in the other. I just haven't got there yet. Mostly looking for directional approval on this PR change this close to release. And make sure I don't fubar the redirects too :)

Reply:
@howardjohn when I submit the new version of this PR can you resolve this issue if you are satisfied with the last statement I made?

Cheers
-steve

Reply:
PR https://github.com/istio/istio.io/pull/7267 is meant to resolve this concern. For the moment, resolving conversation.